src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,61 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, V> {
+    private T firstObject;
+    private V secondObject;
 
+    public Pair(T firstObject, V secondObject) {
+        this.firstObject = firstObject;
+        this.secondObject = secondObject;
+    }
+
+    public T getFirst() {
```

Comment:
```suggestion
    public T getFirstObject() {
```
the same with setters